Translations/fr.po

```diff
@@ -7354,334 +7354,324 @@ msgctxt "spell"
+#: Source/stores.cpp:275
 msgid "Indestructible,  "
 msgstr "Indestructible, "
 
-#: Source/stores.cpp:277
+#: Source/stores.cpp:283
 msgid "No required attributes"
 msgstr "Aucun attribut requis"
 
-#: Source/stores.cpp:309 Source/stores.cpp:1059 Source/stores.cpp:1295
+#: Source/stores.cpp:315 Source/stores.cpp:1071 Source/stores.cpp:1305
 msgid "Welcome to the"
 msgstr "Bienvenue dans la"
 
-#: Source/stores.cpp:310
+#: Source/stores.cpp:316
 msgid "Blacksmith's shop"
 msgstr "Boutique du forgeron"
 
-#: Source/stores.cpp:311 Source/stores.cpp:665 Source/stores.cpp:1061
-#: Source/stores.cpp:1119 Source/stores.cpp:1297 Source/stores.cpp:1309
-#: Source/stores.cpp:1322
+#: Source/stores.cpp:317 Source/stores.cpp:677 Source/stores.cpp:1073
+#: Source/stores.cpp:1131 Source/stores.cpp:1307 Source/stores.cpp:1319
+#: Source/stores.cpp:1332
 msgid "Would you like to:"
 msgstr "Souhaitez-vous:"
 
-#: Source/stores.cpp:312
+#: Source/stores.cpp:318
 msgid "Talk to Griswold"
 msgstr "Parler à Griswold"
 
-#: Source/stores.cpp:313
+#: Source/stores.cpp:319
 msgid "Buy basic items"
 msgstr "Acheter des objets de base"
 
-#: Source/stores.cpp:314
+#: Source/stores.cpp:320
 msgid "Buy premium items"
 msgstr "Acheter des objets premiums"
 
-#: Source/stores.cpp:315 Source/stores.cpp:668
+#: Source/stores.cpp:321 Source/stores.cpp:680
 msgid "Sell items"
 msgstr "Vendre des objets"
 
-#: Source/stores.cpp:316
+#: Source/stores.cpp:322
 msgid "Repair items"
 msgstr "Réparer des objets"
 
-#: Source/stores.cpp:317
+#: Source/stores.cpp:323
 msgid "Leave the shop"
 msgstr "Quitter la boutique"
 
-#. TRANSLATORS: This text is white space sensitive. Check for correct alignment!
-#: Source/stores.cpp:360 Source/stores.cpp:725 Source/stores.cpp:1096
+#: Source/stores.cpp:372 Source/stores.cpp:737 Source/stores.cpp:1108
 msgid "I have these items for sale:"
 msgstr "J'ai ces objets à vendre:"
 
-msgid "Your gold: {:d}"
-msgstr "Votre or: {:d}"
-
-#. TRANSLATORS: This text is white space sensitive. Check for correct alignment!
-#: Source/stores.cpp:425
+#: Source/stores.cpp:437
 msgid "I have these premium items for sale:"
 msgstr "J'ai ces objets premium à vendre:"
 
-#. TRANSLATORS: This text is white space sensitive. Check for correct alignment!
-#: Source/stores.cpp:544 Source/stores.cpp:818
+#: Source/stores.cpp:556 Source/stores.cpp:830
 msgid "You have nothing I want."
 msgstr "Vous n'avez rien pour moi."
 
-#. TRANSLATORS: This text is white space sensitive. Check for correct alignment!
-#: Source/stores.cpp:555 Source/stores.cpp:829
+#: Source/stores.cpp:567 Source/stores.cpp:842
 msgid "Which item is for sale?"
 msgstr "Quel objet est à vendre?"
 
-#. TRANSLATORS: This text is white space sensitive. Check for correct alignment!
-#: Source/stores.cpp:626
+#: Source/stores.cpp:638
 msgid "You have nothing to repair."
 msgstr "Vous n'avez rien à réparer."
 
-#. TRANSLATORS: This text is white space sensitive. Check for correct alignment!
```

Comment:
Just to make sure, those TRANSLATORS comments are no more useful ?

Reply:
This has indeed been corrected and is no longer relevant :)

Reply:
LGTM then!